### DISCUSSION TITLE
How to install gamedig with Deno?

### DISCUSSION THREAD

**Discussion by fyrak1s** [gamedig/node-gamedig#546] [Q&A]
Hey!
I saw gamedig has now deno support!

I'd be curious to check it out, but what are the instructions to install it with deno?

On nodejs it would just be `npm install gamedig`, how does it work on Deno? Thanks :)

**Comment by CosminPerRam** [comment]
Hey, `npm` is a package manager and Deno does not provide one as its just a Javascript runtime.
So, you'd just use npm to install your libraries, then just use Deno to execute your program.
But Deno can also import packages directly from a link, I found [this medium article](https://medium.com/deno-the-complete-reference/dependency-management-in-deno-48f1c91ad84d) that's about pretty much all you need to know regarding dependency management in Deno.

**Reply by fyrak1s** [reply]
Oh I see! So I have it now installed using npm. I am using gamedig only through LinuxGSM though (I am not directly controlling it). Will gamedig now automatically use Deno if it is installed on the system or how does it work? Ty!
